modules/acquia_cms_search/src/Plugin/AcquiaCmsTour/AcquiaConnectorForm.php

```diff
@@ -38,6 +40,32 @@ protected function getEditableConfigNames() {
     return ['acquia_connector.settings'];
   }
 
+  /**
+   * Acquia Connector Subscription service.
+   *
+   * @var \Drupal\acquia_connector\Subscription
+   */
+  protected $subscription;
+
+  /**
+   * The site profile.
+   *
+   * @var \Drupal\acquia_connector\SiteProfile\SiteProfile
+   */
+  protected SiteProfile $siteProfile;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
```

Comment:
I think we can rewrite this whole as:
```suggestion
  public static function create(ContainerInterface $container) {
    $this->siteProfile = $container->get('acquia_connector.site_profile');
    $this->subscription = $container->get('acquia_connector.subscription');
    return parent::create($container);
  }
```

Reply:
$this-> reference isn't allowed